### ISSUE TITLE
How are the indicators calculated in the paper? How is the dataset used for validation?

### ISSUE THREAD

**Issue by climberwu** [kirito878/denver#2]
Hello, your work is excellent. I would like to use this as a baseline, but in the experiment, were the training set and test set classified separately for each sequence?
During training and testing, we found that the model was only trained and tested on a single sequence. Were the experimental results averaged after testing all sequences?
Please answer this question. Thank you for your wo
